src/Microsoft.Azure.SignalR.Management/Obsolete/ServiceManager.cs

```diff
@@ -31,8 +31,12 @@ public ServiceManager(IServiceCollection serviceDescriptors)
             _serviceProvider = _services.BuildServiceProvider(); ;
             _restClientFactory = _serviceProvider.GetRequiredService<RestClientFactory>();
             var endpointManager = _serviceProvider.GetRequiredService<IServiceEndpointManager>();
-            _endpoint = new ServiceEndpoint(_serviceProvider.GetRequiredService<IOptions<ServiceManagerOptions>>().Value.ConnectionString);
-            _endpointProvider = endpointManager.GetEndpointProvider(_endpoint);
+            var connectionString = _serviceProvider.GetRequiredService<IOptions<ServiceManagerOptions>>().Value.ConnectionString;
+            if (!string.IsNullOrWhiteSpace(connectionString))
+            {
+                _endpoint = new ServiceEndpoint(connectionString);
+                _endpointProvider = endpointManager.GetEndpointProvider(_endpoint);
```

Comment:
how about check null here for _endpoint  and _endpointProvider ? then no need to check later

Reply:
If check null here, then `ConnectionString = null && ServiceEndpoints!=null` will be invalid, which should be valid.

Reply:
is ServiceEndpoints use here?

Reply:
No